data/hai/hai reveal 5 philosophers.txt

```diff
@@ -873,7 +857,7 @@ mission "Hai Reveal [B12] Devil-Hide Battle"
 			`To complete this mission, you must land with Choot'k, Teeneep, Danforth and Wallace's ships all present in the system and intact...`
 	on enter "Devil-Hide"
 		conversation
-			`Teeneep sends out a message to the fleet informing you that she is engaging the electronic warfare baffling and reminds you that everyone needs to get onto the ground intact if this is to go well. Choot'k's Shield Beetle has red engine flares; the others have blue. So if he gets into trouble you should be able to tell at a distance. The baffling won't work forever, so get to it.`
+			`Teeneep sends out a message to the fleet informing you that she is engaging the electronic warfare baffling and reminds you that everyone needs to get onto the ground intact if this is to go well. In case Choot'k gets into trouble, here's how to recognize his Shield Beetle from a  distance: it is reinforced with a darkened hull and has red engine flares while the others have blue. The baffling won't work forever, so get to it.`
```

Comment:
Given the radical difference in sprite, I think you can safelly assume anyone who isn't blind is going to be able to identify the unique ship. It made some sense to mention it before when the sprite itself was the same and only the engine flares and weapons were different, but I don't think it's necessary to describe all the differences now.
Just say "The General's ship is called the <whatever>, make sure it survives." Or similar.

Reply:
I originally did remove it but Grey didnt agree
Its more distinct now though

Reply:
Do we even need to say anything? It makes sense the general is on the cool special beetle no?

Reply:
Well, if the idea is that the General's ship needs to be saved, it perhaps merits some mention, especially if it is acceptable to lose all the other ships. But, no, I don't think you need to describe the ship in detail when it is so obviously unique. At best, just say, "It's the one that looks different."

Reply:
Its only acceptable to lose the 2 SB